src/tests/playwright/pages/dsp-graph-evaluation.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <script type="module" defer>
+      import { evaluateGraph } from './util/audio-comparer.js';
+      import { test, evaluate } from './util/audit.js';
```

Comment:
Oh, are we importing the entire `audit.js` library?

Reply:
What a coincidence! We also have the same `audit.js` in the official WPT suite. That's why I got confused:
https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/web_tests/webaudio/resources/audit.js